smartgymapi/handlers/cardio_activity.py

```diff
@@ -0,0 +1,80 @@
+import logging
+from datetime import datetime
```

Comment:
meh zo te zien houden we in de andere files bij standard library import ook de style aan van  ```
```
import module
from module import

# third party en local imports
```
Dit vind ik zelf ook leesbaarder.

PEP8 zegt niets over de sorting zolang we standard, third party, local maar aanhouden en de rest wel alfabetisch.

dusseh move along :)